### ISSUE TITLE
'Upload' Button for Attaching Structure XML Not Working in Safari

### ISSUE THREAD

**Issue by joncameron** [avalonmediasystem/avalon#4334]
### Description
For some users, the Upload button for Structure files (under "edit?step=structure" for a media object) is not able to be selected/pressed. I couldn't reproduce this, but I did see strange behavior relating to the cursor (shown in the gif below) where it isn't changing to the hand icon as it does for the other buttons.

### Environment
Safari: 14.0.1
OS: Mojave 10.14.6
Site: Media Collections Online (Production)

### Example
![Dec-07-2020 14-19-25](https://user-images.githubusercontent.com/1406618/101395046-6293c280-3897-11eb-8b76-6cb901844136.gif)

**Comment by jlhardes** [comment]
I am experiencing the same problem in Safari 14.0.1. The Upload button is not functioning, even when I try clicking on it after moving the mouse around it to get the hand pointer.

**Comment by glwhicks** [comment]
I am using  Safari is 14.0.1. My OS is Mojave 10.14.6. The Upload button is not working. The hand pointer does not appear.

**Comment by Dananji** [comment]
Since Browserstack only offers Safari 13.1 as the latest version I tested this there and the file upload function is working (but it doesn't change the pointer and button color when hovering like for the other buttons).
And I did some research on this, and found the following;
According to the responses in [this thread](https://discussions.apple.com/thread/252051818) in apple website, this seems to be an issue with this specific version of Safari. 
The solutions suggested are as follows;
- from the user side: to either downgrade Safari or to use a different browser until this gets fixed.
- from the application side: to implement drag-n-drop functionality for the file upload which doesn't work with this UI, or change the file upload input form (I'm exploring this option)

**Comment by Dananji** [comment]
I see the cursor and hover color change not happening in previous Safari versions in Browserstack (Tested on Safari 10.1 on macOS Sierra). Since I cannot test this in Safari 14.0.1, I tried to fix the button appearance and cursor using a different file upload form. What I tried was introducing a hidden file upload input into the form and then using that within the `Upload/Replace` button. This still doesn't change the pointer and hover color change.
